lib/rust/ensogl/core/src/display/world/stats.rs

```diff
@@ -30,19 +30,19 @@ impl {
         let performance      = web::performance();
         let mut monitor      = debug::Monitor::new();
         let panels = vec![
-            monitor.add( debug::monitor::FrameTime          :: new(&stats) ),
-            monitor.add( debug::monitor::Fps                :: new(&stats) ),
-            monitor.add( debug::monitor::WasmMemory         :: new(&stats) ),
-            monitor.add( debug::monitor::GpuMemoryUsage     :: new(&stats) ),
-            monitor.add( debug::monitor::DrawCallCount      :: new(&stats) ),
-            monitor.add( debug::monitor::DataUploadCount    :: new(&stats) ),
-            monitor.add( debug::monitor::DataUploadSize     :: new(&stats) ),
-            monitor.add( debug::monitor::BufferCount        :: new(&stats) ),
-            monitor.add( debug::monitor::SymbolCount        :: new(&stats) ),
-            monitor.add( debug::monitor::ShaderCount        :: new(&stats) ),
-            monitor.add( debug::monitor::ShaderCompileCount :: new(&stats) ),
-            monitor.add( debug::monitor::SpriteSystemCount  :: new(&stats) ),
-            monitor.add( debug::monitor::SpriteCount        :: new(&stats) ),
+            monitor.add( debug::monitor::FrameTime          :: default() ),
+            monitor.add( debug::monitor::Fps                :: default() ),
+            monitor.add( debug::monitor::WasmMemory         :: default() ),
+            monitor.add( debug::monitor::GpuMemoryUsage     :: default() ),
+            monitor.add( debug::monitor::DrawCallCount      :: default() ),
+            monitor.add( debug::monitor::DataUploadCount    :: default() ),
+            monitor.add( debug::monitor::DataUploadSize     :: default() ),
+            monitor.add( debug::monitor::BufferCount        :: default() ),
+            monitor.add( debug::monitor::SymbolCount        :: default() ),
+            monitor.add( debug::monitor::ShaderCount        :: default() ),
+            monitor.add( debug::monitor::ShaderCompileCount :: default() ),
+            monitor.add( debug::monitor::SpriteSystemCount  :: default() ),
+            monitor.add( debug::monitor::SpriteCount        :: default() ),
```

Comment:
In such a case, this API should be changed to use turbo fish syntax instead of doing defaults everywhere. Passing things as values make sense if there is any case when the values change. In case the values do not change, pass them as types instead.